.github/ISSUE_TEMPLATE.md

```diff
@@ -16,7 +16,7 @@ $ openshift-install version
 
 # Platform:
 <!--
-Please specify below either aws, libvirt or openstack below.
+Please specify the platform type: aws, libvirt, openstack, baremetal, or none (UPI)
```

Comment:
Looks good. git short log should say "none" as well and while you're at it, no need for '.' in shortlog :) Might be good to update the PR title as per new changes as well.